pkg/k8s/client.go

```diff
@@ -32,23 +33,25 @@ var NatsGVK = schema.GroupVersionResource{
 
 //go:generate go run github.com/vektra/mockery/v2 --name=Client --outpkg=mocks --case=underscore
 type Client interface {
```

Comment:
nit: can we have either all parameters in the signature named or none? some functions still use unnamed parameters (getsecret, patchapply)

Reply:
updated